frontend/src/component/feature/StrategyTypes/FlexibleStrategy/FlexibleStrategy.tsx

```diff
@@ -77,18 +81,18 @@ const FlexibleStrategy = ({
             return defaultStickiness;
         }
 
-        return parseParameterString(parameters.stickiness);
+        return parseParameterString(parameters.stickiness || defaultStickiness);
     }, [loading, parameters.stickiness]);
 
     if (parameters.stickiness === '') {
         onUpdate('stickiness')(stickiness);
     }
 
     useEffect(() => {
-        if (isDefaultStrategyEdit && !parameters.groupId) {
-            onUpdate('groupId')('');
+        if (!parameters.groupId) {
+            onUpdate('groupId')(isDefaultStrategyEdit ? '' : featureId);
```

Comment:
Does this handle the case where you've emptied the field yourself?

Reply:
Yes. It will show an error 

![image](https://github.com/user-attachments/assets/69592bc6-3529-47c8-8be1-39f82004d48c)

Reply:
Cool!